### PR TITLE
create factories and seeder

### PR DESCRIPTION
**Pull request by dungvan**

### CODE REVIEW

File: database/factories/ModelFactory.php [hunk 1]
```diff
@@ -16,9 +16,82 @@
...
+
+$factory->define(App\Order::class, function (Faker\Generator $faker) {
+    return [
+        'user_id' => $faker->randomElement(App\User::pluck('id')->toArray()),
+        'created_at' => $faker->dateTime,
+        'updated_at' => $faker->dateTime,
+        'trans_at' => $faker->dateTime,
+        'custom_address' => $faker->paragraph,
+        'payment' => 10000,
+        'status' => random_int(0, 3)
```
Comment (ghost) [inline database/factories/ModelFactory.php:77]: -why did you define it is boolean?

File: database/factories/ModelFactory.php [hunk 2]
```diff
@@ -16,9 +16,82 @@
...
+    ];
+});
+
+$factory->define(App\OrderItem::class, function (Faker\Generator $faker) {
+    $rand = random_int(0, 1);
+    if ($rand == 1) {
+        return [
+            'target_id' => $faker->randomElement(App\Food::pluck('id')->toArray()),
```
Comment (ghost) [inline database/factories/ModelFactory.php]: -update column name with morphy column.

File: database/factories/ModelFactory.php [hunk 3]
```diff
@@ -16,9 +16,82 @@
...
+            'target_type' => 'food',
+            'quantity' => random_int(1, 5)
+        ];
+    }
+    else {
+        return [
+            'target_id' => $faker->randomElement(App\Material::pluck('id')->toArray()),
```
Comment (ghost) [inline database/factories/ModelFactory.php]: -update column name with morphy column.

**Review by ghost — APPROVED** [review]
check comment. please

**Review by ghost — CHANGES_REQUESTED** [review]
check comment. please.

**Review by ghost — CHANGES_REQUESTED** [review]
Check comment, please!

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment. thank you!

File: database/factories/ModelFactory.php (hunk 1 above)
Comment (dungvan) [reply database/factories/ModelFactory.php:77]: i was changes this to tinyInteger in migration! thank you!

Comment (ghost) [reply database/factories/ModelFactory.php:77]: ok. thank you!

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**